meta-rdk-mtk-bpir4/recipes-wifi/hostapd/files/hostapd-init-EHT.sh

```diff
@@ -0,0 +1,257 @@
+#!/bin/sh
+
+mac80211_add_capabilities() {
+	local __var="$1"; shift
+	local __mask="$1"; shift
+	local __out= oifs
+
+	oifs="$IFS"
+	IFS=:
+	for capab in "$@"; do
+		set -- $capab
+
+		[ "$(($4))" -gt 0 ] || continue
+		[ "$(($__mask & $2))" -eq "$((${3:-$2}))" ] || continue
+		__out="$__out[$1]"
+	done
+	IFS="$oifs"
+
+	export -n -- "$__var=$__out"
+}
+
+gen_vht_cap() {
```

Comment:
Where does the code in this routine come from?  It is definitely not original, and may need credit.